test/builder_hyperone.bats

```diff
@@ -18,7 +18,7 @@ USER_VARS="${USER_VARS} -var token=${HYPERONE_TOKEN}"
 USER_VARS="${USER_VARS} -var project=${HYPERONE_PROJECT}"
 
 hyperone_has_image() {
-    h1 image list --project-select=${HYPERONE_PROJECT} --query "[?tag.${2}=='${3}']"  --output=tsv | grep $1 | wc -l
+    h1 image list --project-select=${HYPERONE_PROJECT} --query "[?tag.${2}=='${3}']"  --output=tsv | grep $1 -c
```

Comment:
Do you want to add yourself as CODEOWNER for this as well?

Reply:
Yes, see #9259 .